src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -114,12 +114,13 @@ export const ARShowScreen = ({ navigation, route }) => {
 
 const objectParser = async ({ item, setObject, setIsLoading, onPress }) => {
   const parsedObject = { texture: [] };
+  const localUris = item?.payload?.scenes?.[0]?.localUris;
```

Comment:
can you please add a TODO here please for the 0 and our future idea to handle this somehow based on other logics?

Reply:
Yes, I added that!
https://github.com/ikuseiGmbH/smart-village-app-app/pull/667/commits/0234e9b239abd5f44871528c7c1e3f8bf29be70d